src/app/bookings/bookings.component.html

```diff
@@ -16,7 +25,7 @@
     <!-- Weight Column -->
     <ng-container matColumnDef="departureTime">
       <th mat-header-cell *matHeaderCellDef> Departure Time </th>
-      <td mat-cell *matCellDef="let element"> {{element.flight.departureTime}} </td>
+      <td mat-cell *matCellDef="let element"> {{dateFormated(element.flight.departureTime)}} </td>
```

Comment:
We are implementing this in different ways, lets get together to figure out which one to use